web/book/src/README.md

```diff
@@ -1,4 +1,4 @@
-# PRQL language book
+# PRQL Language Reference Book
```

Comment:
One of the sections is called "Reference", so I don't think it makes sense to name the whole book "Reference book"...

Reply:
OK